sdk/tables/azure-data-tables/azure/data/tables/_table_client.py

```diff
@@ -247,43 +249,41 @@ def delete_entity(
     @distributed_trace
     def create_entity(
             self,
-            entity,  # type: Union[TableEntity, dict[str,str]]
+            entity,  # type: Union[TableEntity, Dict[str,str]]
             **kwargs  # type: Any
     ):
-        # type: (...) -> TableEntity
+        # type: (...) -> Dict[str,str]
         """Insert entity in a table.
 
         :param entity: The properties for the table entity.
         :type entity: Union[TableEntity, dict[str,str]]
-        :return: TableEntity mapping str to azure.data.tables.EntityProperty
-        :rtype: ~azure.data.tables.TableEntity
+        :return: Dictionary mapping response headers from the service
```

Comment:
We probably should avoid documenting this as "response headers" - maybe calling it something like "operation metadata"